packages/docs/src/pages/en/components/cards.md

```diff
@@ -110,6 +110,52 @@ Cards can be colored by using any of the builtin colors and contextual names usi
 
 <example file="v-card/prop-color" />
 
+#### Elevation
+
+The **elevation** property provides up to 24 levels of shadow depth. By default, cards rest at 2dp.
+
+<example file="v-card/prop-elevation" />
+
+#### Hover
+
```

Comment:
Could use a description
(side note, `hover` prop seems to be bugged)

Reply:
Exactly. I thought about describing it as on the `v-card` API or as on the `v-button` docs. But I don't know how the final behavior is supposed to be when fixed. I guess it should start with some elevation and adds 2?

Reply:
describing it similar to api is fine
However looking at the sass vars on this, it looks like base elevation is actually elevation=1, and hover is to apply elevation=8